docs/configuration.md

```diff
@@ -312,7 +313,7 @@ In some cases the host value is not present in an event (or an IP address is in
 who require a true hostname be attached to each event, SC4S provides an optional facility to perform a reverse IP to
```

Comment:
that requires a true hostname will be attached to each event, SC4S provides an optional facility to perform a reverse IP to